docs/tutorials/Bonding_and_Charge/mulliken_population.md

```diff
@@ -56,4 +56,130 @@ Si              2     1.356   2.644   0.000   0.000   4.000     0.000
+
+For both of these cases, we will use an identical ```.param``` file (just rename them to ```diamond.param``` and ```GaAs.param```). The cell files will be slightly different - for diamond we will change the ```lattice_abc``` block lattice dimensions with 2.52 (rather than 3.8 - the structure is the same but the cell size is different). Naturally the ```Si```'s in the ```positions_frac``` block need to be replaced with ```C```'s
+
+Towards the end of ```diamond.castep``` we find
+
+```
+Atomic Populations (Mulliken)
+-----------------------------
+Species          Ion     s       p       d       f      Total   Charge (e)
+==========================================================================
+C               1     1.076   2.924   0.000   0.000   4.000     0.000
+C               2     1.076   2.924   0.000   0.000   4.000     0.000
+==========================================================================
+
+            Bond                   Population      Length (A)
+======================================================================
+          C 1 -- C 2                    3.00        1.54318
+======================================================================
+```
+The results are rather similar but there are a couple of interesting things to note:
+
+- The ratio of the populations in s and p orbitals is closer to 1:3, which is expected for sp^3^ hybridization - this indicates that the bonds are much more perfectly hybridized, meaning it's more overlapped: this indicates stronger bonding, as well as showing that silicon is a semimetal
+- The population of electrons in the bonds is the same (indicating that the same type of bonding is present), but the bond length is smaller - again indicating more overlap and thus stronger bonding.
+
+Now we will compare it with GaAs. The same procedure is used, except the lattice length is now 3.93 and the atoms in ```positions_frac``` should be ```Ga``` and ```As``` (it doesn't matter which one goes in which position/line).
+
+We get an output looking like this:
+
+```
+Atomic Populations (Mulliken)
+-----------------------------
+Species          Ion     s       p       d       f      Total   Charge (e)
+==========================================================================
+Ga              1     1.180   1.743   9.994   0.000  12.917     0.083
+As              1     1.488   3.595  10.000   0.000  15.083    -0.083
+==========================================================================
+
+            Bond                   Population      Length (A)
+======================================================================
+         Ga 1 -- As 1                   0.34        2.44652
+======================================================================
+```
+
+Despite having the same structure, the results are very different now:
+
+- Unlike before, the Ga and As ions have charges on them - this is indicative of ionic/polar character
```

Comment:
I was putting the others element symbols in verbatim because I was directly referencing how they're written in the code block, just to make it absolutely clear I was talking about changing the file code. Not sure if that's the best way to approach it/how others do that